.github/workflows/custom_helpline.yml

```diff
@@ -0,0 +1,120 @@
+# This is a basic workflow to build and upload Webchat to S3 bucket with Github Actions
+
+name: Custom Helpline Release
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
+
+  workflow_call:
+    inputs:
+      account-sid:
```

Comment:
If you do the changes above this is not relevant, but if you don't, this should be named & described to clearly indicate it is the name of an SSM parameter containing the account SID, not the account SID itself